src/scripts/mainApp.js

```diff
@@ -75,6 +75,8 @@ function msToTime(duration) {
     return hours + ":" + minutes + ":" + seconds
 }
 
+// FIXME: why are you requiring ipc renderer each time even after you already stored a reference in "var ipc"?
+// FIXME: never assigned soundPlayer...
```

Comment:
Since soundPlayer is in the main.html file you can directly access it through js by using it's ID (This is not safe and will be replaced with a variable)